app/lib/gh-docs/repo-content.ts

```diff
@@ -17,13 +18,25 @@ export async function getRepoContent(
 ): Promise<string | null> {
   if (ref === "local") return getLocalContent(filepath);
   let [owner, repo] = repoPair.split("/");
-  let pathname = `/${owner}/${repo}/${ref}/${filepath}`;
-  let response = await fetch(
-    new URL(pathname, "https://raw.githubusercontent.com/").href,
-    { headers: { "User-Agent": `docs:${owner}/${repo}` } },
-  );
-  if (!response.ok) return null;
-  return response.text();
+  let contents = await octokit.rest.repos.getContent({
```

Comment:
This is probably fine to pull in `octokit` directly. However, I noticed that in [some other files](https://github.com/remix-run/remix-website/blob/main/app/lib/gh-docs/tags.ts#L16_ we pass `octokit` in as an argument. Do you know if there was any specific reason for this pattern initially?

Reply:
i don't know if there was any reason for passing it, looks like chance made that decision sometime last year before he left. i can match the pattern if preferred.